Tests/AppTests/__Snapshots__/WebpageSnapshotTests/test_AuthorShow.1.html

```diff
@@ -4,19 +4,19 @@
     <meta name="robots" content="noindex"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <meta charset="UTF-8"/>
-    <meta name="og:site_name" content="The Swift Package Index"/>
+    <meta property="og:site_name" content="The Swift Package Index"/>
```

Comment:
I'm a bit baffled by this upstream change in Plot:

https://github.com/JohnSundell/Plot/compare/0.12.0...0.13.0#diff-72971a1e6abae9162a5d0cf6be214779684aace8b0b67b9d2818793caf31b27a

`<meta>` doesn't mention `property` specifically:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta